src/content/admin/cloud-credentials/gcp.mdx

```diff
@@ -20,10 +20,20 @@ GCP credentials are configured using [Workload Identity Federation](https://clou
 
 As a first step let's create a Workload Identity Pool. A Workload Identity Pool is a group of identities that can be used to access GCP resources.
 
+Okteto displays the OIDC endpoint of your cluster in the Admin -> General view of the Dashboard:
+
+  <p align="center">
+     <Image
+       src={require("@site/static/img/amazon-ecr-oidc-config.png").default}
+       alt="OIDC configuration in admin general view"
+     />
+   </p>
+
 ```bash
 PROJECT_ID=my-project-12345
 POOL_ID=okteto-pool
 AUDIENCE=deploy.okteto.dev
+OIDC_ENDPOINT=https://container.googleapis.com/v1/projects/my-project-12345/locations/us-central1/clusters/development
```

Comment:
I believe it’s better to use a variable to make it clear that this is a value that needs to be configured.